src/__test__/items.test.js

```diff
@@ -0,0 +1,53 @@
+import updateStatusItem from '../updateStatus';
```

Comment:
- [ ] You did excellent work except for a single unfortunate error. `updateStatusItem` is not a default export component. As a result, the test run is throwing errors. You can easily fix it by destructuring it (put the component in curly branches like `{ updateStatusItem }`. Kindly fix it and you are good to go. :pray: 

![Screenshot from 2021-10-02 01-34-23](https://user-images.githubusercontent.com/64894927/135677153-f0b724c1-0fcf-4cdf-be6e-1f488c45e58f.png)